com.getpostman.Postman.yml

```diff
@@ -0,0 +1,46 @@
+app-id: com.getpostman.Postman
+base: io.atom.electron.BaseApp
+base-version: stable
+runtime: org.freedesktop.Platform
+runtime-version: 1.6
+sdk: org.freedesktop.Sdk
+command: Postman
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --filesystem=home
+modules:
+  - name: Postman
+    buildsystem: simple
+    build-commands:
+      - tar -xf Postman-*.tar.gz
+      - mkdir -p /app/Postman
+      - mv Postman/app/* /app/Postman
+      - install -Dm644 /app/Postman/resources/app/assets/icon.png /app/share/icons/hicolor/128x128/apps/com.getpostman.Postman.png
+      - install -Dm755 run.sh /app/bin/Postman
+      - install -Dm644 com.getpostman.Postman.desktop /app/share/applications/com.getpostman.Postman.desktop
+      - install -Dm644 com.getpostman.Postman.appdata.xml /app/share/metainfo/com.getpostman.Postman.appdata.xml
+    sources:
+      - type: file
+        only-arches:
+          - x86_64
+        url: https://dl.pstmn.io/download/version/6.1.3/linux64
+        sha256: 6e08dae1524416396921337dd785e84436c1887a7d835b24c69ea68dce13290d
+        dest-filename: Postman-linux-x64-6.1.3.tar.gz
```

Comment:
You could give this a generic name (e.g. `dest-filename: Postman.tar.gz`), so you don't have to update that too (and remove the glob above).

Reply:
Why do you even use `type: file` and not `type: archive`?

Reply:
Try with `archive` and you'll see. It fails on my machine.

Reply:
https://github.com/flatpak/flatpak-builder/issues/158